Bio/SeqAnalysis/SeqAnalyzer.py

```diff
@@ -100,27 +107,49 @@ def main_calculations(id_seq_dictionary: dict) -> tuple:
         5. amino_acids_percent - of ints and/or floats
         6. instability - of ints and/or floats
         7. flexibility - of ints and/or floats
-        8. secondary_structure_fraction - of tuples (of ints and/or floats) with 3 elements
+        8. secondary_structure_fraction - of dicts with three keys/values with ints and/or floats as values
         9. seq_gravy - of ints and/or floats
-        10. mol_ext_coefficient - of ints and/or floats
-
-        :Example:
-        >>> SeqAnalyzer.main_calculations({'protein1': 'WKQTNSLEGKQ', 'protein2': 'WKQQTNSLEGKQ'})
-        [1318.4349, 1446.5641000000003],
-        [8.59112548828125, 8.59112548828125],
+        10. mol_ext_coefficient - of dicts with two keys/values with ints and/or floats as values
+
+        :Example:   #need to finish
+        >>> SeqAnalyzer.main_calculations({'protein1': 'WKQTNSTDLHEYTLEGKQ', 'protein2': 'TDLHEYTWKQQTNSLEGKQ'})
+        ([2178.314, 2306.443], [5.445, 5.414],
+        [{'A': 0, 'C': 0, 'D': 1, 'E': 2, 'F': 0, 'G': 1, 'H': 1, 'I': 0, 'K': 2, 'L': 2, 'M': 0, 'N': 1, 'P': 0,
+        'Q': 2, 'R': 0, 'S': 1, 'T': 3, 'V': 0, 'W': 1, 'Y': 1}, {'A': 0, 'C': 0, 'D': 1, 'E': 2, 'F': 0, 'G': 1,
+        'H': 1, 'I': 0, 'K': 2, 'L': 2, 'M': 0, 'N': 1, 'P': 0, 'Q': 3, 'R': 0, 'S': 1, 'T': 3, 'V': 0, 'W': 1, 'Y': 1}]
+        ,[0.111, 0.105], [{'A': 0.0, 'C': 0.0, 'D': 5.556, 'E': 11.111, 'F': 0.0, 'G': 5.556, 'H': 5.556, 'I': 0.0,
+        'K': 11.111, 'L': 11.111, 'M': 0.0, 'N': 5.556, 'P': 0.0, 'Q': 11.111, 'R': 0.0, 'S': 5.556, 'T': 16.667,
+        'V': 0.0, 'W': 5.556, 'Y': 5.556}, {'A': 0.0, 'C': 0.0, 'D': 5.263, 'E': 10.526, 'F': 0.0, 'G': 5.263,
+        'H': 5.263, 'I': 0.0, 'K': 10.526, 'L': 10.526, 'M': 0.0, 'N': 5.263, 'P': 0.0, 'Q': 15.789, 'R': 0.0,
+        'S': 5.263, 'T': 15.789, 'V': 0.0, 'W': 5.263, 'Y': 5.263}], [17.928, 19.737], [1.005, 1.009],
+        [{'Helix': 22.222, 'Turn': 16.667, 'Sheet': 22.222}, {'Helix': 21.053, 'Turn': 15.789, 'Sheet': 21.053}],
+        [-1.661, -1.758], [{'Cysteines_reduced': 6990, 'Cysteines_residues': 6990},
+        {'Cysteines_reduced': 6990, 'Cysteines_residues': 6990}])
         """
         list_of_proteinanalysis_objects = [ProteinAnalysis(seq) for seq in id_seq_dictionary.values()]
-        molecular_weight = [seq.molecular_weight() for seq in list_of_proteinanalysis_objects]
-        pi = [seq.isoelectric_point() for seq in list_of_proteinanalysis_objects]
+        molecular_weight = [round(seq.molecular_weight(), 3) for seq in list_of_proteinanalysis_objects]
+        pi = [round(seq.isoelectric_point(), 3) for seq in list_of_proteinanalysis_objects]
```

Comment:
Same as above, it's more convinient if instead of use `3` you'll use `self.rounding` variable which is configurable in `def __init__`